spinn_front_end_common/utility_models/reverse_ip_tag_multi_cast_source.py

```diff
@@ -200,8 +200,7 @@ def send_buffer_times(self, send_buffer_times):
         self._send_buffer_times = send_buffer_times
         for (vertex_slice, vertex) in self._machine_vertices:
             send_buffer_times_to_set = self._send_buffer_times
-            if (self._send_buffer_times is not None and
-                    len(self._send_buffer_times) > 0):
+            if self._send_buffer_times is not None and self._send_buffer_times:
```

Comment:
send_buffer_times can definitely be a numpy array (I just tested using a spike source array with a numpy array, and this fails)

Reply:
Is it _always_ a numpy array or just sometimes?

Reply:
Just sometimes - when a user uses a numpy array vs. a list.  Unfortunately you won't know which it might be...

Reply:
Ugh. OK.